app/components/forms/events/view/order-form.js

```diff
@@ -0,0 +1,30 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        orderExpiryTime: {
+          identifier : 'orderExpiryTime',
+          rules      : [
+            {
+              type   : 'integer[1..60]',
+              prompt : this.get('l10n').t('Please enter a valid registration time limit between 1 to 60 minutes.')
+            }
+          ]
+        }
+      }
+    };
+  },
+  actions: {
+    submit(data) {
+      this.onValid(() => {
+        this.sendAction('save', data);
```

Comment:
instead of this use closure actions like in rest of the app.

Reply:
@srv-twry hmm, maybe open an issue and change them all, `this.sendAction` is really not required  if we are following DDAU approach like in most of the ember tables.